cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -711,61 +767,108 @@ pub mod tests {
             check_build_metadata(&flox, &env_metadata, &env, &builder, EXAMPLE_PACKAGE_NAME)
                 .unwrap();
 
-        let publish_provider = PublishProvider::<&MockCache> {
+        let publish_provider = PublishProvider {
             build_metadata,
             env_metadata,
-            cache: None,
         };
 
-        let res = publish_provider.publish(&client, &catalog_name).await;
+        reset_mocks(&mut flox.catalog_client, vec![Response::Publish(
+            PublishResponse { ingress_uri: None },
+        )]);
+
+        // TODO: meta-only
+        let res = publish_provider
+            .publish(&flox.catalog_client, &catalog_name, None, None, false)
```

Comment:
Is the TODO here that the last arg should be `true`? What's blocking that?

Reply:
That was a stale comment, I added that before adding the `reset_mocks()` call above